tools/redis/redis-cluster-1.conf

```diff
@@ -0,0 +1,5 @@
+port 7001
```

Comment:
I don't know if this is the right/best place to put this - those files are the redis cluster config files used by `docker-compose.yml`.

Reply:
Cool, I've renamed it to `redis-tests`.